### PR TITLE
Implemented ES6 Feature to Awesome Books Project

### PR DESCRIPTION
**Pull request by AyoMoses1**
I implemented ES6 features to the awesome books project and also maintained and followed best practices for the Html, CSS and JavaScript files. Thanks

### CODE REVIEW

File: modules/controller.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { listOfBooks } from './selectors.js';
+
+const displayBooks = (booksArray) => {
+  const collection = JSON.parse(localStorage.getItem('books'));
+  booksArray.books = collection;
+
+  booksArray.books.forEach((book) => {
```
Comment (rloterh) [inline modules/controller.js]: - [ ] I'm unable to add or remove books due to app throwing an error in runtime from the browser console, as can be seen below. `bookArray` appears to be null when the program starts as it does not hold any book initially. I suggest that you try wrapping your enumerable around a conditional block.

**Load Error:**
image

File: README.md [hunk 2]
```diff
@@ -1 +1,210 @@
...
+  - Yes, there will certainly be a desktop version to aid desktop users in having a friendly UX experience on the website
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (rloterh) [inline README.md:207]: - [ ] OPTIONAL
MIT license link throws a 404-page request error when visited. Kindly endeavor to fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ayo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃

## Highlights 👍  
- Good job so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @AyoMoses1**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- No Linter errors :heavy_check_mark: 
- The app is working fine, you can add and remove books from the list corretly  ✔️  
- The current date is displayed below the navigation bar using the Luxon library 🥇 
- Module is imported `import { DateTime } from "luxon";` 👍🏻 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**